js/ui/radio_group/radio_group.js

```diff
@@ -255,9 +257,12 @@ var RadioGroup = Editor.inherit({
         return this._$submitElement;
     },
 
-    _contentReadyHandler: function() {
+    _contentReadyHandler: function(e) {
+        this._radios = this._radios || e.component;
```

Comment:
For what?

Reply:
RadioGroup uses an instance of RadioCollection. And I see that you keep on storing `this._radios`. I believe that if you want to store something( e.g.  `this._radios`) the best place to save it when you create it.

Reply:
In your case at the first time `_contentReadyHandler` will be fired before `this._radios` storing. If I understand you right.